### ISSUE TITLE
No errors in non-empty error message

### ISSUE THREAD

**Issue by WalterS** [janfri/multi_exiftool#15]
After reading a directory, multi_exiftool's error contains "17214 image files read", but no actual error.
```ruby
ARGV.each do |dir|
  r = ''
  r = '**' if @options[:recursive] == true
  allfiles = Dir.glob(File.join(File.expand_path(dir), r, '*.orf'))
  next if allfiles.empty?

  results, errors = MultiExiftool.read(allfiles)
  
  unless errors.empty?
      p errors
      raise('Reading exifs failed')
  end
end
```
In irb, I went over each file in that directory and got no error:
```ruby
allfiles = Dir.glob(File.join(File.expand_path(dir), r, '*.orf'))
allfiles.each do |f|
  results, errors = MultiExiftool.read([f])
   
  unless errors.empty?
    p errors
    raise("Reading #{f} failed")
  end
end
```

This error occurs only in one directory, in the other eight I worked on multi_exiftool behaves as expected.

**Comment by janfri** [comment]
It's a problem of the actual way of parsing the error messages. I hope to fix this in the next days.

**Comment by janfri** [comment]
Which version of ExifTool are you using?

**Comment by janfri** [comment]
It should be fixed in multi_exiftool 0.18.0. It was necessary to adapt the parsing of the error messages. Therefore ExifTool version 11.10 or higher is required now.

Please let me know if it works.

**Comment by janfri** [comment]
I will close this issue now. Feel free to reopen it if you have questions or multi_exiftool 0.18.0 doesn't solve your problem.
